src/containers/LtiModel/index.js

```diff
@@ -0,0 +1,259 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from "react-redux";
+import { Accordion, Card, Button } from "react-bootstrap";
+import Swal from "sweetalert2";
+import { withRouter } from "react-router-dom";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import {loadMyProjectsltiAction} from "store/actions/project";
+ 
+import "./style.scss" 
+
+const  LTIProjectShared = (props) => {
+ 
+  const [activeProject, setActiveProject] = useState("");
+  const [activePlaylist, setActivePlaylist] = useState("");
+  const [activeactivity, setactiveactivity] = useState("");
+  const [allproject, setAllproject] = useState(null);
+  const [formdata, setformdata] = useState("");
+  const dispatch = useDispatch();
+  const allProjects = useSelector((state) =>  state);
+  const {match} = props
+ 
+  useEffect(() => {
+    dispatch(
+      loadMyProjectsltiAction(
+        decodeURIComponent(match.params.lms_url),
+        decodeURIComponent(match.params.lti_client_id)
+      )
+    );
+  }, []);
+
+  useEffect(() => {
+    allProjects.project.projects && setAllproject(allProjects.project.projects);
```

Comment:
Please do not use this syntax, instead use if condition
I think I mentioned it before

Reply:
updated